docs/reference/tasking-reference.md

```diff
@@ -1,7 +1,227 @@
+
+<h3> Returns </h3>
+
+| Type   | Description       |
+| ------ | ----------------- |
+| `dict` | Order parameters. |
+
+<h3> Example </h3>
+
+```python
+tasking.construct_order_parameters(
+    data_product_id="123eabab-0511-4f36-883a-80928716c3db",
+    name="PNeo tasking order",
+    acquisition_start="2023-11-01",
+    acquisition_end="2023-12-20",
+    geometry = {
+        "type": "Polygon",
+        "coordinates": (
+            (
+                (13.375966, 52.515068),
+                (13.375966, 52.516639),
+                (13.378314, 52.516639),
+                (13.378314, 52.515068),
+                (13.375966, 52.515068),
+            ),
+        ),
+    }
+)
+```
+
+## Feasibility study
+
+### get_feasibility
+
+The `get_feasibility` function returns a list of feasibility studies for tasking orders.
+
+```python
+get_feasibility(
+    feasibility_id=None,
+    workspace_id=None,
+    order_id=None,
+    decision=None,
+    sortby="createdAt",
+    descending=True
+)
+```
+
+<h3> Arguments </h3>
+
+| Name             | Type                  | Description                                                                                                                           |
+| ---------------- | --------------------- | ------------------------------------------------------------------------------------------------------------------------------------- |
+| `feasibility_id` | `Optional[str]`       | The feasibility study ID.                                                                                                             |
+| `workspace_id`   | `Optional[str]`       | The workspace ID.<br/><br/>Use to get objects from a specific workspace. Otherwise, objects from the entire account will be returned. |
+| `order_id`       | `Optional[str]`       | The order ID.                                                                                                                         |
+| `decision`       | `Optional[list[str]]` | The status of feasibility studies. The allowed values:<ul><li>`NOT_DECIDED`</li><li>`ACCEPTED`</li></ul>                              |
+| `sortby`         | `str`                 | Arrange elements in the order specified in `descending` based on a chosen field.                                                      |
+| `descending`     | `bool`                | Arrange elements in ascending or descending order based on the field specified in `sortby`.                                           |
+
+
+<h3> Returns </h3>
+
+| Type   | Description                    |
+| ------ | ------------------------------ |
+| `JSON` | A list of feasibility studies. |
+
+<h3> Example </h3>
+
+```python
+tasking.get_feasibility(
+    feasibility_id=None,
+    workspace_id="68567134-27ad-7bd7-4b65-d61adb11fc78",
```

Comment:
this is a fake id right?

Reply:
We have a couple of sample IDs that we use in the API reference, it's them